mods/ts/sequences/structures.yaml

```diff
@@ -862,6 +862,10 @@ proc: # TODO: unused narefn_a, narefn_b
 	make: ntrefnmk
 		Length: 20
 		ShadowStart: 20
+	dock: ntrefn_a # TODO: unused
```

Comment:
This might be better played on the harv, as it is the same as our other harv dump/dock anims (but in overlay form). That is another pr, though.